content/articles/Website-vs-Web-app/index.md

```diff
@@ -0,0 +1,131 @@
+
+Website is of different types. Some of them are:
+
+1. Dating websites
+
+2. Educational Websites
+
+3. Social Media websites
+
+4. Portfolio website
+
+#### Characteristics of a Website
+A good website developer can examine a site according to its features. One of the characteristics that will determine the success of a website is its appearance. Let's have a look at some additional website attributes.
+1. `User friendly`: A website should be easy to operate.
+2. `Attractive and accessible for all targeted users`:This helps attract users to opt for a particular website against another. 
+3. ``Purpose``- What is the objective of the website's appearance? A good website should provide a purpose and that purpose should be clear to the user within a short period. A user should not take much of his/her time trying to understand the reason for the appearance of a site.
+   The unclear purpose may lead to a website not giving back good results.
+4. `High speed of loading`: Web developers should consider this feature as not all end-users are patient. Therefore if a website loads slowly, a user will most likely look for a different website with a high speed. There numerous elements that contribute to the loading speed of a website. The most crucial being the hosting quality. Therefore, always choose a host that loads fast.
+5. `Secure`- A good website should provide security for both end-users and the website developers.
+
+#### Importance of a website
+
+1. ``It provides self-service customer support in Business``- If your website support Q&A post, then you can provide a forum where user can have questions about your goods among themselves. Other users can provide answers to the questions.
+
+   This can make things a lot quicker for your clients and make your client service office more effective.
+
+2. `Professionalism` : Studies show that most people opt to deal with businesses with websites compared to those that lack one. 
+
+3. ``It gives opportunities abroad``- If you have a website, you create an opportunity for everyone worldwide to find you. If you're selling your product online, you end up having more market compared to a business person selling locally. This will in turn increase the profit. 
+
+### Web application
+A web application is a program that can be accessed via a browser and is linked to a database. A web application contains both functional and interactive elements.
+Unlike websites, web apps are more complex to create (because they are very customized) and require an experienced team.
+Below are examples of web application:
+- [Twitter](https://twitter.com/?lang=en)
+- [Gmail](https://mail.google.com/)
+- [Wikipedia](https://www.wikipedia.org/)
+
+#### Characteristics of A web Application
+
+1. ``Accessibility``- A good web app is available and usable by a maximum number of people. The web application should be accessible to people with disabilities.
```

Comment:
I like how you have explained these points💯. 
Good job. 👍  
I think if you do the importance of a website like this, we will be good to go.